web/src/components/pages/dashboard/stats/contribution-activity.tsx

```diff
@@ -1,36 +1,23 @@
 import * as React from 'react';
```

Comment:
Basically rewrote this. [Split diff view](?diff=split&w=1) is easier to see whats changed.